### PR TITLE
Med statement changes mk

### PR DESCRIPTION
**Pull request by markkramerus**
Proposed changes for medication statement to focus on cancer-related medications.

### CODE REVIEW

File: input/fsh/CancerRelatedMedicationStatement.fsh [hunk 1]
```diff
@@ -8,7 +8,7 @@ It is important to note that cancer-related medications are not the ONLY medicat
 
 Conformance statement:
 
-MedicationStatement resources associated with an mCODE patient that are prescribed to treat the cancer condition SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
+MedicationStatement resources associated with an [mCODE patient](conformance.html#mCODE-Patients) SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
```
Comment (masnick) [inline input/fsh/CancerRelatedMedicationStatement.fsh]: I think the 2nd sentence can be removed -- it seems to me like it's redundant with the first sentence given we're saying all of the base resource SHOULD conform.

```suggestion
MedicationStatement resources associated with an [mCODE patient](conformance.html#mCODE-Patients) SHOULD conform to this profile."
```

Comment (markkramerus) [reply input/fsh/CancerRelatedMedicationStatement.fsh]: It is actually not redundant. 

The first statement covers MedicationStatement resources.

The second statement covers the case where a different resource is used for something that reasonably ought to have been a MedicationStatement.

File: input/fsh/CancerRelatedMedicationStatement.fsh [hunk 2]
```diff
@@ -16,10 +18,15 @@ MedicationStatement resources associated with an [mCODE patient](conformance.htm
 6) informationSource is missing PractitionerRole
 7) not sure why reasonCode has be changed from 0..* to 0..1
 */
-* effective[x] and medication[x] MS
+* subject 1..1 MS
+* subject only Reference(CancerPatient)
```
Comment (mlterryMitre) [inline input/fsh/CancerRelatedMedicationStatement.fsh:22]: If we're removing references to CancerPatient, then why even bother having a CancerPatient profile?

**Review by mlterryMitre — COMMENTED** [review]
I don't agree with the changes, but I suppose this content will also change as we work through other review cycles and iterations to all the medication-related profiles.
We also still have to add clear documentation on how MedicationStatement will differ from MedicationRequest and MedicationAdministration for usage if the decision is made to keep MedicationStatement.

File: input/fsh/CancerRelatedMedicationStatement.fsh [hunk 3]
```diff
@@ -2,7 +2,9 @@ Profile:  CancerRelatedMedicationStatement
 Parent:   MedicationStatement
 Id:       mcode-cancer-related-medication-statement
 Title:    "Cancer-Related Medication Statement"
-Description:    "A record of the use of a medication (individual administration or entire course) for a condition that is related to a primary or secondary cancer condition. The use may be reported by the patient or clinician and administration does not have to be directly observed.
+Description:    "A record of the use of a medication (individual administration or entire course) intended to treat cancer. Cancer-related medications are identified by the reasonCode or reasonReference, and include both curative and palliative medications. The use may be reported by the patient or clinician and administration does not have to be directly observed. The profile includes two extensions, TreatmentIntent and TerminationReason, that capture additional context around the use of the medication.
```
[the remark this answers is not in the document]
Comment (masnick) [reply input/fsh/CancerRelatedMedicationStatement.fsh:5]: [In the draft mCODE Bundle](https://github.com/HL7/fhir-mCODE-ig/pull/27/files#diff-87ede991f6486344e56eae80de9c0fa33acbb481c0cc7000f41d4b9edd69e4c8R22-R33), I said all medications should be included, not just ones conforming to an mCODE profile. My assumption is that only a few (if any) medication resources for a given patient would use the extra elements we add in the mCODE profiles, and that there’s no way to accurately pre-determine which medications are relevant for a given use case.

My question is: can inclusion of supportive medications fall back to being pulled into the mCODE Patient Bundle, or do we need to do something here as May says?

**Comment by markkramerus** [comment]
Closing the PR because we have made the decision to use MedicationRequest to be consistent with US Core.

**Pull request closed without merging**